From 1c88a9ae6e12d6cba334a3646c7f634bff109cf9 Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Thu, 6 Apr 2023 17:34:23 -0400
Subject: [PATCH] [WIP] pinenote: Declare panel orientation

This orientation assumes "book reading" orientation with the power
button at the top, and the USB port at the bottom.

The panel_reflection=1 orientation assumes landscape layout in the cover
case.

---
 arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi b/arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi
index 65d28fac69985..52b854483bcaa 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi
@@ -123,6 +123,9 @@ led-0 {
 
 	panel {
 		compatible = "eink,ed103tc2";
+		// XXX when using panel_reflection=0
+		// NOTE: using panel_reflection=0 since it matches the touchscreen orientation.
+		rotation = <90>;
 
 		port {
 			panel_in_ebc: endpoint {
-- 
2.38.0

